modules/oadp-1-3-1-performance-kubevirt.adoc

```diff
@@ -0,0 +1,281 @@
+// This module is included in the following assembly:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-performance/oadp-1-3-performance.adoc
+
+[id="oadp-1-3-1-performance-kubevirt_{context}"]
+= {oadp-short} 1.3.1 Kubevirt Coverage
+
+The objective of this test is to test {oadp-first} Backup and restore namespaces with idle big VMS with restic/velero-csi plugins to external NVME storage.
+
+
+[id="oadp-1-3-1-kubevirt-single-cluster_{context}"]
+== Single cluster backup and restore
+
+[width="100%",cols="10%,15%,7%,8%,6%,16%,15%,7%,16%",options="header",]
+|===
+|Backup Location
+|Cloud Provider
+|Volumemode
+|Backup Type
+|vm-image
+|Data check method
+|OpenShift-Virt version
+|OADP build
+|Status
+
+|MinIO
+|Google Cloud Platform (GCP)
+|Filesystem
+|Data Mover (dm)
+|Fedora
+|single large single file
+|4.14
+|1.3.1
+|Pass
+
+|MinIO
+|GCP
+|Block
+|dm
+|Fedora
+|single large single file
+|4.14
+|1.3.1
+|Pass
+
+|MinIO
+|GCP
+|Block
+|dm
+|Fedora
+|todolist app
+|4.15
+|1.3.1
+|Pass
+
+|MinIO
+|GCP
+|Block
+|dm
+|windows
+|single small file
+|4.14
+|1.3.1-35
+|Pass
+
+|MinIO
+|GCP
+|Filesystem
+|dm
+|windows
+|single small file
+|4.14
+|1.3.1-35
+|Pass
+
+|MinIO
+|GCP
+|Block
+|dm
+|rhel9
+|single large single file
+|4.15
+|1.3.1
+|Pass
+
+|MinIO
+|GCP
+|Block
+|dm
+|Fedora
+|single large single file
+|4.15
+|1.3.1
+|Pass
+
+|Multi Cloud Gateway (MCG)
+|GCP
+|Filesystem
+|dm
+|Fedora
+|single large single file
+|4.15
+|1.3.1
+|*resource issues*
+
+|MCG
+|GCP
+|Block
+|dm
+|Fedora
+|single large single file
+|4.15
+|1.3.1
+|*resource issues*
+
+|MinIO
+|PSI (cnv pipeline)
+|Block
+|csi & dm
+|Fedora
+|single large single file
+|4.15
+|1.3.1
+|Pass
+
+|MinIO
+|PSI (cnv pipeline)
+|Block |csi & dm
+|Fedora
+|single large single file
+|4.15
+|1.3.1
+|Pass
+
+|GCP
+|GCP
+|Block
+|csi & dm
+|Fedora
+|single large single file
+|4.15
+|1.3.1
+|Pass
+
+|GCP
+|GCP
+|Filesystem
+|csi & dm
```

Comment:
DM ( datamover) is always first a CSI backup.  I see some notes through out that have "dm"  and "csi & dm"  This is the same thing